src/data/index.js

```diff
@@ -0,0 +1,15 @@
+import data from "./data";
```

Comment:
May we use TypeScript?

Reply:
Of course, but i didn't know how ?

Reply:
You don't have to, just rename your file to `*.tsx` and use TS ;)

Reply:
ok